src/konserve/utils.cljc

```diff
@@ -21,30 +21,24 @@
     {:key key :type type :last-write (now)}
     (clojure.core/assoc old :last-write (now))))
 
+#?(:clj
 (defmacro async+sync
   [sync? async->sync async-code]
   (let [async->sync (if (symbol? async->sync)
-                      (resolve async->sync)
-                      async->sync)
-        res
-        (if (boolean? sync?)
```

Comment:
This branch inlines the decision and avoids emitting the other branch if the sync keyword is directly given. Was it necessary to remove this functionality?

Reply:
Anyway, not that important. We can do it like this and expand the macro later again if it makes sense. Right now this static expansion is not usable anyway.